TestTools/StreamChatTestTools/Fixtures/JSONs/Poll.json

```diff
@@ -0,0 +1,34 @@
+{
+    "poll": {
+        "id": "7fd88eb3-fc05-4e89-89af-36c6d8995dda",
+        "name": "test",
+        "description": "",
+        "voting_visibility": "public",
+        "enforce_unique_vote": false,
+        "max_votes_allowed": null,
+        "allow_user_suggested_options": false,
+        "allow_answers": false,
+        "vote_count": 0,
+        "options": [],
+        "answers_count": 0,
+        "vote_counts_by_option": null,
+        "latest_votes_by_option": {},
+        "own_votes": [],
```

Comment:
We are not covering the Votes and Options parsing it seems?

Reply:
added